components/public-api-server/pkg/apiv1/oidc.go

```diff
@@ -301,6 +301,9 @@ func dbOIDCClientConfigToAPI(config db.OIDCClientConfig, decryptor db.Decryptor)
 			AuthorizationEndpoint: decrypted.RedirectURL,
 			Scopes:                decrypted.Scopes,
 		},
+		OidcConfig: &v1.OIDCConfig{
```

Comment:
Noticed `oidcConfig.issuer` wasn't returned in our responses for clients. This returns it, but would like to make sure it wasn't intentional that we weren't returning it.

Reply:
It was probably overlooked, because it wasn't used anywhere. Thanks for adding it 🙏🏻